examples/android_local_test/src/test/java/com/example/bazel/MainActivityTest.java

```diff
@@ -0,0 +1,25 @@
+package com.example.bazel;
+
+import android.app.Activity;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.android.controller.ActivityController;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Junit Test using Robolectric with AssertJ matchers.
+ */
+@RunWith(AndroidJUnit4.class)
```

Comment:
This should be RobolectricTestRunner.class

Reply:
4.0 docs recommend AndroidJUnit4. RobolectricTestRunner was for 3.0. http://robolectric.org/blog/2018/05/09/robolectric-4-0-alpha/